APIs/API_TitleMenus/API_TitleMenus.py

```diff
@@ -62,6 +63,9 @@ def on_run_process_input(self):
 	cur_menu.process_input(self)
 	new_state = self.state
 
+	if new_state == prev_state and not cur_menu.blocks_char_sheet_and_examine:

```

Comment:
This is the esc fix right? I think a comment here would be really helpful

Reply:
nope, this handles scrolling the panel, i've added a comment where esc fix is handled.